build.psm1

```diff
@@ -2808,6 +2808,25 @@ assembly
     }
 }
 
+$psOptionsPath = 'psoptions.json'
+
+function Save-PSOptions {
+    $Script:Options = Get-PSOptions
```

Comment:
no need to use the script scop in a variable used entirely inside a function

Reply:
Fixed.